src/Sylius/Bundle/CoreBundle/Formatter/AppliedPromotionInformationFormatter.php

```diff
@@ -30,6 +30,9 @@ public function format(CatalogPromotionInterface $catalogPromotion): array
         /** @var string $code */
         $code = $catalogPromotion->getCode();
 
-        return [$code => $translationLabels];
+        return [$code => [
+            'isExclusive' => $catalogPromotion->isExclusive(),
```

Comment:
I would be for a snake case notation for keys in array:
```suggestion
            'is_exclusive' => $catalogPromotion->isExclusive(),
```